scvi/external/scbasset/_model.py

```diff
@@ -192,6 +203,193 @@ def get_cell_bias(self) -> np.ndarray:
         """
         return self.module.cell_bias.cpu().numpy()
 
+    @classmethod
+    def _download_motifs(cls, genome: str, motif_dir: str) -> None:
+        """Download a set of motifs injected into peaks"""
+        logger.info(f"Downloading motif set to: {motif_dir}")
+        # download the motif set
+        url_name = cls.MOTIF_URLS.get(genome, None)  # (url, dir_name)
+        if url_name is None:
+            raise ValueError(f"{genome} is not a supported motif set.")
+        urlretrieve(url_name[0], filename=Path(motif_dir, f"{genome}_motifs.tar.gz"))
```

Comment:
we have an internal download function we can use here:

https://github.com/jacobkimmel/scvi/blob/a3ac84016ad2f30e481edd944c1cf65e2f00a4cd/scvi/data/_download.py#L13-L14

Reply:
switched